game-core/src/main/java/games/strategy/engine/data/export/GameDataExporter.java

```diff
@@ -116,11 +116,8 @@ private static String playertechs(final GameData data) {
           for (final TechAdvance tech : frontier.getTechs()) {
             String name = tech.getName();
             final String cat = tech.getProperty();
-            for (final String definedName : TechAdvance.ALL_PREDEFINED_TECHNOLOGY_NAMES) {
-              if (definedName.equals(name)) {
-                name = cat;
-                break;
-              }
+            if (TechAdvance.ALL_PREDEFINED_TECHNOLOGY_NAMES.contains(name)) {
```

Comment:
Nice :+1: 
I wonder/suspect this code might predate Java 1.5 and the collections API